internal/config/config.go

```diff
@@ -280,6 +280,8 @@ func (c *Config) GetRegistryName() string {
 }
 
 // Validate performs validation on the configuration
+//
+//nolint:gocyclo
```

Comment:
Is this necessary, or is it possible to refactor to reduce cyclomatic complexity?